docs/llama.cpp.md

```diff
@@ -9,7 +9,7 @@ llama.cpp is the best backend in two important scenarios:
 
 #### Pre-converted
 
-Download the ggml model directly into your `text-generation-webui/models` folder, making sure that its name contains `ggml` somewhere and ends in `.bin`. It's a single file.
+Download the ggml or gguf models directly into your `text-generation-webui/models` folder, making sure that its name contains `ggml` or `gguf` somewhere and ends in `.bin`. It's a single file.
```

Comment:
I think that gguf models end in .gguf

Reply:
Yup, I just pulled a model from hugginface to confirm. I've updated this.